std_msgs/Float64MultiArray lat
std_msgs/Float64MultiArray lon
std_msgs/Float64MultiArray speed
std_msgs/Float64MultiArray delay